docs/api/asyncio_client.rst

```diff
@@ -557,8 +557,8 @@ transaction), so we have to redo all the work done.
 Generally it's recommended to not execute any long running
 code within the transaction unless absolutely necessary.
 
-Transactions allocate expensive server resources and having
-too many concurrently running long-running transactions will
+Transactions allocate expensive server resources, and having
+too many concurrently long-running transactions will
```

Comment:
This looks good with one minor tweak to make it sound a bit more natural:

```suggestion
too many concurrent long-running transactions will
```

Reply:
Yes it's. Thank you